DOC-331 channel clarification (#380)
amplitude/amplitude-docs: content/collections/data/en/channels.md

To create a channel, follow these steps:

1. Navigate to the *Properties* section of Amplitude Data and open the *Channels* tab.
2. Click *+ Add Channel Classifier.* A pre-built, default channel definition screen opens. Optionally, click the default channel title (*Channel*) to edit the name. You can also add a description below it, if you like.
3. To begin creating the definition of the channel, start from the default template or click *Clear Table* to clear the table's contents.  
  
    ![image2.gif](/docs/output/img/data/image2-gif.gif)  
      
    You can add multiple properties to a row to create a more complex classifier.

4. Add one row for each channel you would like to define. For example, you can add rows labeled `Paid`, `Organic`, `Referral`, and `Direct` to create a high-level channel definition.

> 5. Fill the values for each row. Each cell must evaluate to True for an event to classify to that channel.

Reviewer: ```suggestion
5. Fill the values for each row. Each cell must evaluate to `True` for an event to classify to that channel.
```

Reviewer: Values should be formatted as code, I think.

    If a channel could be defined as `row A OR row B`, you can add one row for A and one row for B, and then set the channel name to be the same for both rows.

> 6. Click *Save*. Amplitude labels channels in property drop downs under the *Channels* category.

Reviewer: ```suggestion
6. Click *Save*. Amplitude labels channels in property dropdowns under the *Channels* category.
```

Reviewer: In accordance with the new directive on formatting this term.

## Compare metrics between channels with Data Tables

> Amplitude’s [Data Tables](/docs/analytics/charts/data-tables/data-tables-multi-dimensional-analysis) allow you to define metrics critical to your bottom line, such as CPA, AOV and ROAS. Evaluate how these metrics perform between different channels by **grouping by your channel** on the left-hand column. To see how these channels break down further, add other core dimensions like campaigns.

Reviewer: ```suggestion
Amplitude’s [Data Tables](/docs/analytics/charts/data-tables/data-tables-multi-dimensional-analysis) allow you to define metrics critical to your bottom line, such as CPA, AOV, and ROAS. Evaluate how these metrics perform between different channels by **grouping by your channel** on the left-hand column. To see how these channels break down further, add other core dimensions like campaigns.
```

Reviewer: We use the Oxford comma around here.

![image1.png](/docs/output/img/data/image1-png.png)

Amplitude attributes metrics in different ways if a user has multiple touch points with different channels before converting. Amplitude has built [attribution modeling](/docs/analytics/charts/data-tables/data-tables-attribute-credit) into data tables to enable defining user attribution in your channels.

{{partial:admonition type="tip" heading=""}}
Applying a channel classifier before an attribution model can attribute values in potentially unexpected ways. For example, in a sequence consisting of:  
  
`CPC --> email --> website`  
  
Applying a channel classifier of `CPC only` results in `email` and `website` being filtered out. In this case, the attribution model would attribute to CPC.
{{/partial:admonition}}

## Use cases

> * **Blended views:** Easily create top level blended views of all paid and all organic traffic to easily see how efficiency and performance has changed over time.

Reviewer: ```suggestion
* **Blended views:** Easily create top level blended views of all paid and all organic traffic to easily see how efficiency and performance have changed over time.
```

> * **High-level channels:** You can break down your core metrics according to the common definitions of channels you already use in [Google Analytics](https://support.google.com/analytics/answer/6010097?hl=en#zippy=%2Cin-this-article) and [Adobe](https://experienceleague.adobe.com/docs/analytics/components/marketing-channels/c-getting-started-mchannel.html?lang=en).

Reviewer: ```suggestion
* **High-level channels:** Break down your core metrics according to the common definitions of channels you already use in [Google Analytics](https://support.google.com/analytics/answer/6010097?hl=en#zippy=%2Cin-this-article) and [Adobe](https://experienceleague.adobe.com/docs/analytics/components/marketing-channels/c-getting-started-mchannel.html?lang=en).
```
